workflows/eager.nf

```diff
@@ -333,7 +343,27 @@ workflow EAGER {
         ch_versions = ch_versions.mix( PRESEQ_LCEXTRAP.out.versions )
     }
 
-     //
+
+    //
+    // MODULE: Bedtools coverage 
+    //
+    
+    if ( params.run_bedtools_coverage ) {
+        ch_dedupped_bams.combine(ch_anno_for_bedtools).map{
+            meta, bam, bai, anno ->
+            [meta, anno, bam]
+        }.set { ch_dedupped_for_bedtools }
```

Comment:
Our convention is not to use set (as it hides away the channel name when trying to rapidly find how each section of the pipeline links together via channels.). Another convention is each operator goes on a new line